nightguard WatchKit Extension/InterfaceController.swift

```diff
@@ -86,11 +86,25 @@ class InterfaceController: WKInterfaceController, WKCrownDelegate {
     override func willActivate() {
         super.willActivate()
         
-        guard WKExtension.shared().applicationState == .active else {
-            return
-        }
+//        guard WKExtension.shared().applicationState != .background else {
+//            return
+//        }
         
         isActive = true
+        
+        let currentNightscoutData = NightscoutCacheService.singleton.getCurrentNightscoutData()
+        updateInterface(withNightscoutData: currentNightscoutData, error: nil)
+        
+        delay(0.4) { [unowned self] in
```

Comment:
Could you add a comment here, why there has to be delayed here? Looks a bit like a hack ;-)

Reply:
Unfortunately I cannot add the comments myself, for the next two weeks I’ll be away from my laptop, in  family vacation. 
We can go on with the pull request when I’ll be back, better. But we can discuss things meanwhile, you can check the features, the code, list the bugs (it was under heavy development last week, I’m sure there are - I just find some myself :)).

Reply:
Yes - I always have to turn my wrist to get the first update on the watch. I will add a comment that we can remove this, if apple fixes that problem...